<ui version="4.0" >
 <class>bcPropertyDialog</class>
 <widget class="QDialog" name="bcPropertyDialog" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Boundary conditions</string>
  </property>
  <widget class="QTabWidget" name="bcTab" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>10</y>
     <width>311</width>
     <height>151</height>
    </rect>
   </property>
   <property name="tabShape" >
    <enum>QTabWidget::Triangular</enum>
   </property>
   <property name="currentIndex" >
    <number>1</number>
   </property>
   <widget class="QWidget" name="heatEquation" >
    <attribute name="title" >
     <string>Heat equation</string>
    </attribute>
    <widget class="QLabel" name="temperatureLabel" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Temperature</string>
     </property>
    </widget>
    <widget class="QLabel" name="heatFluxLabel" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Heat flux</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="temperatureEdit" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>10</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="heatFluxEdit" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab" >
    <attribute name="title" >
     <string>Navier-Stokes</string>
    </attribute>
    <widget class="QLabel" name="Velocity1Label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Velocity 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="Velocity2Label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Velocity 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="Velocity3Label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>Velocity 3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Velocity1Edit" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>10</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Velocity2Edit" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Velocity3Edit" >
     <property name="geometry" >
      <rect>
       <x>80</x>
       <y>50</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="linearElasticity" >
    <attribute name="title" >
     <string>Linear elasticity</string>
    </attribute>
    <widget class="QLineEdit" name="displacement1Edit" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>10</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="displacement2Edit" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="displacement3Edit" >
     <property name="geometry" >
      <rect>
       <x>100</x>
       <y>50</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="displacement1Label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Displacement 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="displacement2Label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Displacement 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="displacement3Label" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text" >
      <string>Displacement 3</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="applyButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>170</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text" >
    <string>Apply</string>
   </property>
   <property name="icon" >
    <iconset resource="../Mesh3D.qrc" >:/icons/list-add.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="discardButton" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>170</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Discard</string>
   </property>
   <property name="icon" >
    <iconset resource="../Mesh3D.qrc" >:/icons/list-remove.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Mesh3D.qrc" />
 </resources>
 <connections/>
</ui>
